static/js/util/api.js

```diff
@@ -1,4 +1,4 @@
-// @flow
+// @flow weak
```

Comment:
I'm not sure if changing from `?string` to `string` is a good idea. Anyone can still PATCH the API to make these fields `null` so for some user these fields may end up being `null`. I'm not sure if we should be using flow to describe our types as they should be or to describe all possible edge cases. What do you think?